### ISSUE TITLE
Incorrect -A account arguments causes IndexError

### ISSUE THREAD

**Issue by ddietz89** [olcf/olcf-test-harness#23]
Other conditions may cause this too, I did not extensively test.

```
sbatch  -p batch run_hip_test.sh
Traceback (most recent call last):
  File "/ccs/proj/stf016/olcf-test-harness/harness/bin/test_harness_driver.py", line 477, in <module>
    test_harness_driver()
  File "/ccs/proj/stf016/olcf-test-harness/harness/bin/test_harness_driver.py", line 443, in test_harness_driver
    unique_id, jstatus, actions)
  File "/ccs/proj/stf016/olcf-test-harness/harness/bin/test_harness_driver.py", line 123, in auto_generated_scripts
    submit_exit_value = mymachine.submit_batch_script()
  File "/ccs/proj/stf016/olcf-test-harness/harness/machine_types/rhel_x86.py", line 99, in submit_batch_script
    submit_exit_value = self.submit_to_scheduler(batch_script)
  File "/ccs/proj/stf016/olcf-test-harness/harness/machine_types/base_machine.py", line 82, in submit_to_scheduler
    submit_exit_value = self.__scheduler.submit_job(batchfilename)
  File "/ccs/proj/stf016/olcf-test-harness/harness/machine_types/slurm.py", line 69, in submit_job
    jobid = jobid_pattern.findall(records[0])[0]
IndexError: list index out of range
```

The real error from `sbatch` is:

```
sbatch: error: 2 ERRORS FOUND: 
sbatch: error:  
sbatch: error: You are not a member of account xxx. Please specify a valid account with the -A flag. Valid accounts are: 
sbatch: error:     yyy
sbatch: error:     zzz
sbatch: error: No partition mapping exists for this account
sbatch: error:  
sbatch: error: Job not submitted, please update and resubmit batch job.
sbatch: error:  
sbatch: error: Batch job submission failed: Unspecified error
```

**Comment by ddietz89** [comment]
PR #25 should fix this

**Comment by verolero86** [comment]
Closed by #25.
